geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/Transformers.scala

```diff
@@ -601,3 +631,41 @@ class CastFunctionFactory extends TransformerFunctionFactory {
     try { conversion(s) } catch { case e: Exception => default }
   }
 }
+
+class MathFunctionFactory extends TransformerFunctionFactory {
+  override def functions = Seq(add, subtract, multiply, divide)
+
+  def parseDouble(v: Any): Double = {
+    v match {
+      case i: Int => i.toDouble
+      case d: Double => d
+      case f: Float => f.toDouble
+      case t: String => t.toDouble
+      case _ => throw new IllegalArgumentException(s"Invalid unable to parse ${v} to double.")
+    }
+  }
+
+  val add = TransformerFn("add") { args =>
+    var s: Double = 0.0
+    args.foreach(s += parseDouble(_))
+    s
+  }
+
+  val multiply = TransformerFn("multiply") { args =>
+    var s: Double = 1.0
+    args.foreach(s *= parseDouble(_))
+    s
+  }
+
+  val subtract = TransformerFn("subtract") { args =>
+    var s: Double = parseDouble(args(0))
+    args.drop(1).foreach(s -= parseDouble(_))
+    s
+  }
+
+  val divide = TransformerFn("divide") { args =>
+    var s: Double = parseDouble(args(0))
+    args.drop(1).foreach(s /= parseDouble(_))
```

Comment:
Does this work with more than 2 arguments?  If not, it might make more sense to write it as args(0) / args(1).